### DISCUSSION TITLE
List of triggers to start after deployment

### DISCUSSION THREAD

**Discussion by ChrisTeuben** [Azure-Player/azure.datafactory.tools#89] [Show and tell]
I miss the ability to specify per environment which triggers should be started automatically after a deployment.
It would be nice if this could be specified in a configuration file per environment.
This creates the possibility to create triggers to be enabled only in the target environment. In this way, the Azure DevOps CI / CD process can be performed without manual work afterwards.

**Comment by NowinskiK** [comment]
Chris, it IS possible to declare the desired state of trigger in the configuration file per environment.

**Reply by ChrisTeuben** [reply]
Oops, shame on me... I'll read the docs again with the right glasses on. :-)

**Comment by ChrisTeuben** [comment]
I still cannot find an example in the documentation but after some tries I got it working with this line in my _config-Production.csv_:
_trigger,ProductionTrigger,runtimeState,"Started"_

The trigger "ProductionTrigger", which is disabled on the source side, is now enabled on the target ("Production") side after a deployment.
I love it!

**Reply by NowinskiK** [reply]
I'm glad that this meets your expectation :)
